packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -70,12 +70,28 @@ export class Accordion extends FoundationElement {
         this.$emit("change");
     };
 
+    private findExpandedItem(): AccordionItem | null {
+        for (let item: number = 0; item < this.accordionItems.length; item++) {
+            if (this.accordionItems[item].hasAttribute("expanded")) {
+                return this.accordionItems[item] as AccordionItem;
+            }
+        }
+        return null;
+    }
+
     private setItems = (): void => {
+        if (this.accordionItems.length === 0) {
```

Comment:
We should also be zeroing out the values set by this function to match the "no items" state.  For example, this.activeid and this.activeItemIndex.